server/service/src/pack_unit.rs

```diff
@@ -0,0 +1,178 @@
+use std::collections::HashMap;
+
+use repository::{
+    EqualFilter, PackUnitRow, PackUnitRowRepository, RepositoryError, StockLineRowRepository,
+    StockOnHandFilter, StockOnHandRepository, StorageConnection,
+};
+
+#[derive(Debug, Eq, PartialEq, Ord, PartialOrd)]
+pub struct ItemPackUnit {
+    pub item_id: String,
+    pub most_used_pack_unit_id: String,
+    pub pack_units: Vec<PackUnitRow>,
+}
+
+// For a particular store, this method returns all pack units grouped by item_id and a reference to the most
+// used pack unit for each item (see ItemPackUnit return type).
+pub fn get_pack_units(
```

Comment:
Please use `///`` slashes to make it a doc comment. For example, then the line shows when hovering where the function is used...

Reply:
[185233d](https://github.com/openmsupply/open-msupply/pull/2357/commits/185233d72fa194dc1a26ebc6bf518b62ac5ec18b) XD